README.rst

```diff
@@ -0,0 +1,145 @@
+learning-assistant
+#############################
+
+|pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
+|license-badge| |status-badge|
+
+Purpose
+*******
+
+Plugin for a learning assistant backend, intended for use within edx-platform.
+
+This library contains data models and logic for a platform wide learning assistant.
+
+Dependencies
+************
+In addition to the edx-platform repository in which this library is installed, this plugin also
+leverages the `frontend-lib-learning-assistant`_ as a frontend interface for the learning assistant.
+
+Getting Started
+***************
+
+Developing
+==========
+
+One Time Setup
+--------------
+.. code-block::
+
+  # Clone the repository
+  git clone git@github.com:openedx/learning-assistant.git
+  cd learning-assistant
+
+  # Set up a virtualenv with the same name as the repo and activate it
+  # Here's how you might do that if you have virtualenvwrapper setup.
+  mkvirtualenv -p python3.8 learning-assistant
+
+Every time you develop something in this repo
+---------------------------------------------
+.. code-block::
+
+  # Activate the virtualenv
+  # Here's how you might do that if you're using virtualenvwrapper.
+  workon learning-assistant
+
+  # Grab the latest code
+  git checkout main
+  git pull
+
+  # Install/update the dev requirements
+  make requirements
+
+  # Run the tests and quality checks (to verify the status before you make any changes)
+  make validate
+
+  # Make a new branch for your changes
+  git checkout -b <your_github_username>/<short_description>
+
+  # Using your favorite editor, edit the code to make your change.
+  vim ...
+
+  # Run your new tests
+  pytest ./path/to/new/tests
+
+  # Run all the tests and quality checks
+  make validate
+
+  # Commit all your changes
+  git commit ...
+  git push
+
+  # Open a PR and ask for review.
+
+License
+*******
+
+The code in this repository is licensed under the AGPL 3.0 unless
+otherwise noted.
+
+Please see `LICENSE.txt <LICENSE.txt>`_ for details.
+
+Contributing
+************
+
+Contributions are very welcome.
```

Comment:
We should run through the deltas in [Repo deltas for edX GitHub repos](https://2u-internal.atlassian.net/wiki/spaces/ENG/pages/493453499/Repo+deltas+for+edx+GitHub+repos) to make sure these are appropriate for the `edx` organization. This line jumped out to me, because it's one of the things we're meant to replace.

Reply:
Okay, just updated with the deltas from that doc. Thanks again!